02-RTB/documenti-interni/verbali/2024-11-26.typ

```diff
@@ -0,0 +1,82 @@
+#import "../../../lib/verbale.typ" : *
+
+#show: body => verbale(
+  data: datetime(day:26, month:11, year:2024),
+  tipo: [interno],
+  versioni: (
+    (
+      vers: "0.1.0",
+      date: datetime(day: 26, month: 11, year: 2024),
+      autore: p.marco,
+      verifica: p.sara,
+      descr: "Redazione del contenuto",
+    ),
+  ),
+  stato: [In redazione],
+  presenze: (
+    p.emanuele,
+    p.samuele,
+    p.sara,
+    p.loris,
+    p.marco,
+    p.matteo,
+    p.lorenzo
+  ),
+  odg: [Ottava riunione di _ALimitedGroup_: vengono decisi i ruoli, vengono decisi i tempi limite e la struttura per redigere i prossimi documenti],
+  body,
+)
+
+= Informazioni generali
+
+#inizio-verbale-interno([*virtuale*], [*26 Novembre 2024*], [*14.30*], [*16.30*])
++ Aggiornamenti sui lavori in corso
++ Assegnate scadenze per la redazione dei documenti
++ Vengono assegnati i ruoli sino al giorno 02-12-2024
+
+#pagebreak()
+= Dettaglio riunione
+
+== Aggiornamenti sui lavori in corso
+
+Il gruppo _ALimitedGroup_, in questa settimana, ha speso innumerevoli ore di orologio per redigere ed analizzare in maniera ordinata, pulita e professionale i documenti riguardanti le Norme di Progetto, Piano di Progetto, Glossario e Analisi dei Requisiti. \
```

Comment:
Scriverei: 
Il gruppo _ALimitedGroup_, in questa settimana, ha continuato la redazione e l'analisi  dei seguenti  documenti: Norme di Progetto, Piano di Progetto, Glossario e Analisi dei Requisiti. \
```suggestion
Il gruppo _ALimitedGroup_, in questa settimana, ha speso innumerevoli ore di orologio per redigere ed analizzare in maniera ordinata, pulita e professionale i documenti riguardanti le Norme di Progetto, Piano di Progetto, Glossario e Analisi dei Requisiti. \